ajax/libs/semantic-ui-calendar/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "semantic-ui-calendar",
+  "description": "Calendar module for Semantic UI",
+  "author": "Michael de Hoog (https://github.com/mdehoog/)",
+  "filename": "calendar.min.css",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/mdehoog/semantic-ui-calendar.git"
+  },
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/mdehoog/semantic-ui-calendar.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*"
+        ]
+      }
+    ]
+  },
+   "keywords": [
```

Comment:
Just reminder, there indent is wrong here, should be one less space before `"keyword"`